lte/cloud/go/services/lte/obsidian/handlers/handlers_test.go

```diff
@@ -1411,6 +1411,7 @@ func TestListAndGetGateways(t *testing.T) {
 	ctx := test_utils.GetContextWithCertificate(t, "hw1")
 	test_utils.ReportGatewayStatus(t, ctx, models.NewDefaultGatewayStatus("hw1"))
 
+	checkedInRecentlyFalse := lteModels.GatewayCheckedInRecently(false)
```

Comment:
I think you are using the same `checkedInRecentlyFalse`  on multiple object.  Even this is a test this can create issues if you ever change this value in one single instance. 

better if you use it like this: 
```
CheckedInRecently: lteModels.GatewayCheckedInRecently(false)
```
Besides, it looks better with the rest ;)

Reply:
The problem is that `CheckedInRecently` is a pointer and I don't think Go allows you to create a variable and return the address in a one-liner. But you're right that using the same object for all these instances is problematic. I'll refactor it.

Reply:
Extracted a function to get the address in one line.